lib/setup/TokenRules.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const AbiBinProvider = require('../AbiBinProvider');
+const Deployer = require('../../utils/deployContract');
+
+const ContractName = 'TokenRules';
+
+/**
+ * Helper method which performs deployment of Token Rules contract.
+ */
+class TokenRules {
+  constructor(auxiliaryWeb3, address) {
```

Comment:
documentation for constructor is missing.
Why do we need address parameter here in constructor?

Reply:
Going forward we would have **registerRule** method.So,for constructing contract instance we would need the tokenrules contract address. This is the reason **address** parameter is needed.